karma/makeconf.js

```diff
@@ -62,13 +62,15 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
         ChromeWebRTC: {
           base: 'Chrome',
           flags: [
+            '--no-sandbox',
```

Comment:
Do we still want to remove chrome from the test matrix after this fix?

Reply:
We keep it in the test matrix, we just no longer allow it to fail.